packages/twenty-front/src/modules/object-record/record-board/components/RecordBoard.tsx

```diff
@@ -197,6 +199,7 @@ export const RecordBoard = () => {
         <ScrollWrapper
           contextProviderName="recordBoard"
           componentInstanceId={`scroll-wrapper-record-board-${recordBoardId}`}
+          fullHeight
```

Comment:
@ehconitin Could you see if this modification is relevant with our recent refactor ?

Reply:
I've fix it again, because my changes was causing another issue, but the change of @ehconitin was causing issues in the dropdown menu adding extra blank space, it's why I've added and extra prop.